templates/supertest/put/put.handlebars

```diff
@@ -1,4 +1,4 @@
-  it('should respond with {{length description}}', function(done) {
+  it('should respond with {{length description}}{{#if inputs}}, {{inputs.body.message}}{{/if}}', function(done) {
```

Comment:
I prob didn't dive into this deep enough, but still don't see the point of having a added description when using user-supplied data.. And I can't figure out where inputs.body.message comes from.